docs/geedocs/docsrc/answer/releaseNotes/relNotesGEE5_3_2.rst

```diff
@@ -80,9 +86,24 @@ Release notes: Open GEE 5.3.2
            - Quitting 32-bit Windows portable server in Windows 7 via Internet Explorer does not work

```

Comment:
The headings "Resolved Issues" and "Known Issues" are listed both as headers and as table titles. Please remove the table title

Reply:
Fixed

Reply:
This is also on the "Updated Libraries" table

Reply:
Fixed